src/collections/pages/fr-CA/fr-CA.11tydata.js

```diff
@@ -0,0 +1,10 @@
+"use strict";
+
+const i18n = require("eleventy-plugin-i18n-gettext");
+const config = require("../../../_data/config.json");
+const path = require("path");
+
+module.exports = () => {
+    let locale = path.basename(__dirname);
+    return i18n.enhance11tydata({}, locale, config.languages[locale].dir);
+};
```

Comment:
Is it possible to move this to the collections dir so that we don't have to have duplicates? If not is there another way, or is it just something we'll have to live with?

Reply:
Unfortunately not. The `enhance11tydata` method derives the locale from the directory path and as such needs to be called within the directory: https://github.com/sgissinger/eleventy-plugin-i18n-gettext/blob/663745aca2f14c21e51837988802e96e1107830f/src/i18n.ts#L55